src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -684,19 +685,25 @@ internal sealed override bool IsInterface
             }
         }
 
-        internal override ManagedKind ManagedKind
+        internal override ManagedKind GetManagedKind(ref HashSet<DiagnosticInfo> useSiteDiagnostics)
         {
-            get
+            var managedKind = _flags.ManagedKind;
+            if (managedKind == ManagedKind.Unknown)
```

Comment:
```suggestion
            if (managedKind == ManagedKind.Unknown && _managedKindUseSiteDiagnostics.IsDefault)
```

Reply:
Thsi needs to be `||` not `&&`

Reply:
Should be ```||```

---
In reply to: [460280504](https://github.com/dotnet/roslyn/pull/46114#discussion_r460280504) [](ancestors = 460280504)

Reply:
Made the change, thanks

Reply:
looks like two developers raced to correct this mistake

Reply:
But what order did u read the suggestions?

;)